src/main/java/com/exasol/releasedroid/adapter/github/GitHubException.java

```diff
@@ -16,6 +16,15 @@ public GitHubException(final String message, final Throwable cause) {
         super(message, cause);
     }
 
+    /**
+     * Create a new instance of {@link GitHubException}.
+     *
+     * @param cause exception cause
+     */
+    public GitHubException(final Throwable cause) {
```

Comment:
I'm not sure if it's a good idea to have this constructor without a message. Usually, it think we should add a message that tells the user what did fail.

Reply:
The GitHubException is wrapped into ReleaseException which tells that GitHub release failed and gives a stack trace of the internal error. I think it's a double work to describe the failed place, it should be already described in the GitHub API library.

Reply:
Sure. But I think you can add the information what you were trying to do on a different level.

Example Stack trace:

```
Failed to open pom file of the previous release in order to extract dependency changes.
Failed to open file pom.xml.
```

In that case the 1. message is very valuable since it adds the relevant information.

Reply:
You could also add information about what you were trying to do. The github lib will add the information what exactly went wrong

Reply:
I think users don't need this info. For users, it's enough to see that the error happened during communication with GitHub. For me as a debugging person, it's enough to see the place in the stacktrace + original error message. I still don't think an additional message is useful :) 
And I have an error message for files, as in your example